lib/talkable/resources/offer.rb

```diff
@@ -5,5 +5,46 @@ def self.parse(result_hash)
       offer.claim_links ||= Hashie::Mash.new(result_hash[:claim_links])
       offer
     end
+
+    def advocate_share_iframe(options = {})
+      show_trigger = !options[:ignore_trigger]
+      tag = campaign_tags.first
+      iframe_options = default_iframe_options(tag).merge(options[:iframe] || {})
+      url = show_trigger ? Furi.update(show_url, query: {trigger_enabled: 1}) : show_url
```

Comment:
You may consider using `merge` instead of update here that will cause new query to be merged with existing url query if any.